src/main/java/com/mathworks/ci/RunMatlabCommandBuilder.java

```diff
@@ -114,10 +124,11 @@ private int execMatlabCommand(FilePath workspace, Launcher launcher,
         // Create MATLAB script
         createMatlabScriptByName(uniqeTmpFolderPath, uniqueCommandFile, workspace, listener, envVars);
         ProcStarter matlabLauncher;
+        String options = getStartupOptions() == null ? "" : getStartupOptions().getStartupOptions();

```

Comment:
getStartupOptions().getStartupOptions() reads redundant may be getStartupOptions().getAllOptions() or getOptions()